File: Views/src/components/financial/components/monthly-fixed-expenses-trancking/dto/monthly-fixed-expense-tracking-dto.ts
import { MyUser } from "src/components/authentication/dto/my-user";
import { CompanyDto } from "src/shared/entities-dtos/company-dto";
import { BankAccountDto } from "../../bank-account-cards/dto/bank-account-dto";
import { CardDto } from "../../bank-account-cards/dto/card-dto";
import { PixDto } from "../../bank-account-cards/dto/pix-dto";
import { MonthlyFixedExpenseDto } from "../../monthly-fixed-expenses/dto/monthly-fixed-expense-dto";


export class MonthlyFixedExpenseTrackingDto {
  id: number;
  companyId: number;
  company: CompanyDto;
  userId: number;
  user: MyUser;
  monthlyFixedExpenseId: number;
  monthlyFixedExpense: MonthlyFixedExpenseDto;
  bankAccountId: number;
  bankAccount: BankAccountDto;
  pix: PixDto;
  pixId: number;
  othersPaymentMethods:string;
  card: CardDto;
  cardId: number;
  wasPaid: Date;
  expiration: Date;
  registered: Date = new Date();
  price: number;
  interest: number;
  deleted: boolean;
}

File: Application/Services/Operations/Finances/Dtos/MonthlyExpenses/MonthlyFixedExpenseTrackingDto.cs
using System;
using System.Collections.Generic;
using Application.Services.Operations.Authentication.Dtos;
using Application.Services.Operations.Finances.Dtos.Bank;
using Application.Services.Operations.Finances.Dtos.Enums;
using Application.Services.Operations.Finances.Dtos.Inheritance;
using Application.Services.Operations.Main.Companies.Dtos;
using Domain.Entities.Main.Companies;

namespace Application.Services.Operations.Finances.Dtos
{
    public class MonthlyFixedExpenseTrackingDto: BaseExpenseTrackingDto
    {

        public int MonthlyFixedExpenseId { get; set; }
        public MonthlyFixedExpenseDto MonthlyFixedExpense { get; set; }
     
    }
}